packages/device-utils/src/modelUtils.ts

```diff
@@ -1,6 +1,3 @@
 import { PartialDevice } from './types';
 
-export const getDeviceModel = (device: PartialDevice): 'T' | '1' => {
-    const { features } = device;
-    return typeof features?.major_version === 'number' && features?.major_version > 1 ? 'T' : '1';
-};
+export const getDeviceModel = (device?: PartialDevice) => device?.features?.model || '';
```

Comment:
@mroz22 is it always available an all not deprecated firmwares? Even in bootloader mode? What about to change it together with refactor of the logic for model R?

Reply:
I'd say yes, it is should be available pretty much everywhere. 

but I am against creating a new method for simply accessing a property.

Reply:
so are you against this version of the method (`export const getDeviceModel = (device?: PartialDevice) => device?.features?.model || '';`) or against something else?

Reply:
`getDeviceModel` has 16 occurrences. Probably not a good idea to remove it here.

Reply:
ok